src/app.ts

```diff
@@ -2,11 +2,26 @@ import express from 'express';
 
 import config from 'config';
 import routes from './routes/ninja';
+import { dbCheck } from './db/database';
+import logger from './logger/logger';
```

Comment:
put index file into logger directory to prevent from importing logger/logger.

Reply:
I do not know if I have understood you correctly. I renamed the file with the logger implementation ;)